openapi/components/schemas/RuleActionAdjustReadyToPayout.yaml

```
@@ -0,0 +1,14 @@
+allOf:
+  - $ref: ./RuleAction.yaml
+  - type: object
+    description: Adjust ready to payout.
+    properties:
+      prioritizeActivePaymentInstruments:
+        type: boolean
+        default: false
+        description: If set as `true`, the payment methods associated to the customer's active payment instruments are displayed at the top of the list.
```

nitpick: [boolean fields](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#boolean-fields)
```suggestion
        description: Specifies if payment methods, that are associated with the customer's active payment instruments, are displayed at the top of the list.
```